Dockerfile

```diff
@@ -8,6 +8,8 @@ COPY go.sum .
 RUN go mod download -x
 RUN GO111MODULE=on go install github.com/lu4p/binclude/cmd/binclude
 
+RUN GO111MODULE=on go get github.com/showwin/speedtest-go
```

Comment:
This is unnessecary, go mod download already installs this.

Reply:
Did not know that. Thanks